nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -90,7 +90,8 @@ controller_server:
     min_y_velocity_threshold: 0.5
```

Comment:
before anything else, any API changes need to have update to the `docs/parameters/params.md` file for descriptions and such. Also those same updates should be added to the navigation.ros.org repo's configuration pages for the modified packages.